docs/concepts/jcloud/index.md

```diff
@@ -482,6 +482,219 @@ jc job logs myjob1 -f rich-husky-af14064067
 :width: 90%
 ```
 
+## Deployments
+
+### Deploy
+
+In Jina's idiom, a project is a [Deployment](https://docs.jina.ai/concepts/orchestration/Deployment/), which represents an end-to-end task such as indexing, searching or recommending. In this document, we use "project" and "Deployment" interchangeably.
+
+```{caution}
+When `jcloud` deploys a deployment it automatically appends the following global arguments to the `deployment.yml`, if not present:
+```
+
+```yaml
+jcloud:
+  version: jina-version
+  docarray: docarray-version
+```
+
+#### Single YAML file
+
+A self-contained YAML file, consisting of all configuration at the [Deployment](https://docs.jina.ai/concepts/orchestration/deployment/)-level and [Executor](https://docs.jina.ai/concepts/serving/executor/)-level.
+
+> A Deployment's `uses` must follow the format `jinaai+docker://<username>/MyExecutor` (from [Executor Hub](https://cloud.jina.ai)) to avoid any local file dependencies:
+
+```yaml
+# deployment.yml
+jtype: Deployment
+with:
+  protocol: grpc
+  uses: jinaai+docker://jina-ai/Sentencizer
+```
+
+To deploy:
+
+```bash
+jc deployment deploy ./deployment.yaml
+```
+
+The Deployment is successfully deployed when you see:
+
+```{figure} img/deployment/deploy.png
+:width: 70%
+```
+---
+
+You will get a Deployment ID, say `pretty-monster-130a5ac952`. This ID is required to manage, view logs and remove the Deployment.
+
+As this Deployment is deployed with the default gRPC protocol (feel free to change it to `http`), you can use `jina.Client` to access it:
```

Comment:
```suggestion
Since this Deployment is deployed with the default gRPC protocol (feel free to change it to `http`), you can use `jina.Client` to access it:
```